README.md

```diff
@@ -142,6 +145,54 @@ Below is a comprehensive list of available configuration properties.
 | webhook.projects.<_projectId_>.secret             | N/A                                             | Webhook secret used to validate webhook requests originating from the respective projectId                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
 | webhook.projects.<_projectId_>.skipSignatureCheck | N/A                                             | Boolean to indicate whether the signature should be validated. TODO remove in favor of empty secret.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
 
+### CMAB Configuration Examples
+
+**Complete CMAB Configuration (OPTIMIZELY_CMAB)**:
+```json
+{
+  "enabled": true,
+  "predictionEndpoint": "https://custom-endpoint.com",
```

Comment:
Do we support customization of this in go-sdk? Not sure about the demands for this url customization.